Gruntfile.coffee

```diff
@@ -75,6 +78,7 @@ module.exports = (grunt) ->
         files:
           'dist/droplet-full.js': ['./src/main.coffee']
         options:
+          ignore: ignoredLanguages
```

Comment:
Presumably when LANGUAGE is not set, ignoredLanguages is undefined and coffeescript just handles this all fine?